### ISSUE TITLE
iphone12 导航栏没适配

### ISSUE THREAD

**Issue by JedShiMing** [rilyu/teaset#436]
iphone 12 上高度小了点儿
[
![image](https://user-images.githubusercontent.com/39255696/96858271-9efb9280-1492-11eb-99d1-c42d36d00a4f.png)
](url)

"react-native": "0.63.3",
"teaset": "0.7.4"

**Comment by JedShiMing** [comment]
Theme.js
新增
```
const IPHONE12__WIDTH = 390;
const IPHONE12__HEIGHT = 844;
```

修改isIPhoneX
```
const isIPhoneX = (() => {
  if (Platform.OS === 'web') return false;

  return (
    Platform.OS === 'ios' &&
    ((D_HEIGHT === X_HEIGHT && D_WIDTH === X_WIDTH) ||
      (D_HEIGHT === X_WIDTH && D_WIDTH === X_HEIGHT)) ||
    ((D_HEIGHT === XSMAX_HEIGHT && D_WIDTH === XSMAX_WIDTH) ||
      (D_HEIGHT === XSMAX_WIDTH && D_WIDTH === XSMAX_HEIGHT)) ||
      ((D_HEIGHT === IPHONE12__HEIGHT && D_WIDTH === IPHONE12__WIDTH) ||
        (D_HEIGHT === IPHONE12__WIDTH && D_WIDTH === IPHONE12__HEIGHT))
  );
})();
```

**Comment by JedShiMing** [comment]
fix-0.7.5 resolved
